src/locales/en/common.json

```
@@ -18,9 +18,19 @@
       "add": "Add",
       "delete": "Delete",
       "edit": "Edit",
-      "save": "Save"
+      "save": "Save",
+      "upload": "Upload"
     },
     "disabled": "Disabled, click switch to enable",
+    "info": {
+      "create_time": "Create Time",
```

```suggestion
      "create_time": "Created At",
```

`Time` may be ambiguous